File: problem-solving/beecrowd/1018-cedulas/src/main.c
#include <stdio.h>

int main(){
  unsigned value, quantity, LIST_OF_CEDULES[7] = {100, 50, 20, 10, 5, 2, 1}, i=0;

  scanf("%u",&value);
  printf("%u\n", value);

  for (i=0; i < 7; i++){
    quantity = (unsigned) value / LIST_OF_CEDULES[i];
    value %= LIST_OF_CEDULES[i];
    printf("%u nota(s) de R$ %u,00\n", quantity, LIST_OF_CEDULES[i]);
  }
}

File: problem-solving/beecrowd/1018-cedulas/src/main.rs
use std::io;

fn main() {
    let cedules_array : [u32; 7]= [100, 50, 20, 10, 5, 2, 1];

    let mut value = String::new();
    io::stdin().read_line(&mut value).expect("Didn't read the line.");
    let mut value : u32 = value.trim().parse().unwrap();

    println!("{}", value);
    for cedule in cedules_array.iter(){
        let quantity : u32 = value / cedule;
        value = value % cedule;
        println!("{} nota(s) de R$ {:.2},00", quantity, *cedule);
    }
}
